Test/civl/inductive-sequentialization/BroadcastConsensus.bpl

```diff
@@ -93,23 +89,19 @@ INV_COLLECT_ELIM({:linear_in "broadcast"} pidsBroadcast:[pid]bool, {:linear_in "
 returns ({:pending_async "COLLECT"} PAs:[PA]int, {:choice} choice:PA)
 modifies CH, decision;
 {
+  var {:pool "INV_COLLECT"} k: int;
+
   assert pidsBroadcast == (lambda i:pid :: pid(i)) && pidsCollect == pidsBroadcast;
   assert CH == MultisetEmpty;
 
   CH := (lambda v:val :: value_card(v, value, 1, n));
   assume card(CH) == n;
   assume MultisetSubsetEq(MultisetEmpty, CH);
 
-  havoc decision;
-
-  assume
-  (exists k:int :: {trigger(k)} 0 <= k && k <= n && trigger(k+1) &&
-    // (forall i:pid :: 1 <= i && i <= k ==> decision[i] == max(CH)) &&
-    decision == (lambda i:pid :: if 1 <= i && i <= k then max(CH) else old(decision)[i]) &&
-    PAs == RemainingCollects(k) &&
-    choice == COLLECT(k+1) &&
-    (k < n ==> PAs[COLLECT(n)] > 0) // help for the prover
-  );
+  assume {:add_to_pool "INV_COLLECT", k} {:add_to_pool "INV_COLLECT", k+1} {:add_to_pool "Collect", COLLECT(n)} 0 <= k && k <= n;
```

Comment:
Can `{:add_to_pool "INV_COLLECT", k} {:add_to_pool "INV_COLLECT", k+1}` just be `{:add_to_pool "INV_COLLECT", k, k+1}`?